### PR TITLE
added esConsumes to modules in EventFilter/RPCRawToDigi

### PR DESCRIPTION
**Pull request by JamminJones**
#### PR description:

added esConsumes to 4 modules
#### PR validation:

the code compiles

### CODE REVIEW

**Comment by Dr15Jones** [comment]
This change was made under my supervision.

File: EventFilter/RPCRawToDigi/plugins/RPCTwinMuxDigiToRaw.cc [hunk 1]
```diff
@@ -29,7 +24,10 @@
 #include "EventFilter/RPCRawToDigi/interface/RPCTwinMuxRecord.h"
 
 RPCTwinMuxDigiToRaw::RPCTwinMuxDigiToRaw(edm::ParameterSet const& config)
-    : bx_min_(config.getParameter<int>("bxMin")),
+    : es_tm_link_map_br_token_(esConsumes<RPCAMCLinkMap, RPCTwinMuxLinkMapRcd>()),
```
Comment (Dr15Jones) [inline EventFilter/RPCRawToDigi/plugins/RPCTwinMuxDigiToRaw.cc:27]: This needs the 3rd template argument denoting this will be gotten in the BeginRun

**Comment by Dr15Jones** [comment]
please test

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/30459	d5eff6f
- code changes are in line with the PR description/title
    - the static check points to ```RPCUnpackingModule``` still using es.get; I suppose this will be updated in a separate PR
- jenkins tests pass and comparisons with the baseline show no (relevant differences)

**Comment by silviodonato** [comment]
+1

**Pull request merged**